### DISCUSSION TITLE
add button to form

### DISCUSSION THREAD

**Discussion by azar-maghsoodi** [area17/twill#2241] [Q&A] [answered]
Hello
Can I add a button to my form? I want to add some buttons to my form, for example a button to generate some pdf file for admin or generate meta title from title. is it possible?

**Comment by ifox** [comment, marked as answer]
Hi @azar-maghsoodi, for sure, you can use any html in your blade form file. If you want to use Twill's button style, you can use the `<a17-button>` component.

**Reply by azar-maghsoodi** [reply]
Thank you for the time!
